src/components/homeTabs/GIVfarm.tsx

```diff
@@ -145,6 +153,17 @@ export const TabGIVfarmBottom = () => {
 	const { chainId } = useWeb3React();
 	const [showArchivedPools, setArchivedPools] = useState(false);
 
+	const mainnetGIVStaking = getGivStakingConfig(config.MAINNET_CONFIG);
+	const mainGIVStakeDiscontinued =
+		mainnetGIVStaking.discontinued &&
+		getNowUnixMS() > mainnetGIVStaking.discontinued;
+
+	const xdaiGIVStaking = getGivStakingConfig(config.XDAI_CONFIG);
+
+	const xdaiGIVStakeDiscontinued =
+		xdaiGIVStaking.discontinued &&
+		getNowUnixMS() > xdaiGIVStaking.discontinued;
+
```

Comment:
please move GIVstaking configs to the out of the component, I think the networkProvider.ts is a good place for it.